docker/px4-dev/scripts/entrypoint.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+# Start virtual X server in the background if there is no display
+if [ -z "$DISPLAY"]; then
+	echo "Starting Xvfb"
+	Xvfb :99 -screen 0 1600x1200x24+32 &
```

Comment:
The `1600x1200` was chosen based on other user experiences on the Gazebo/ROS forums. `x24+32` was stolen from #166, but I tested `x16` as recommended by the forum entry I read. Any opinions here are welcome.

Reply:
`99` seems to be the convention for things like this (remote screen use) from the other examples I ran across.